docs/context/ref/ChatRoom.md

```diff
@@ -0,0 +1,64 @@
+---
+id: ChatRoom
+sidebar_label: ChatRoom
+title: Contact
+hide_title: true
+---
+# `ChatRoom`
+
+Reference to the chat room(s) (which could be used later to send a message to the room(s)).
+
+## Type
+
+`fdc3.chat.room`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/chatRoom.schema.json
+
+## Details
+
+| Property    | Type    | Required | Example Value     |
+|-------------|---------|----------|-------------------|
+| `type`      | string  | Yes      | `'fdc3.chat.room'`  |
+| `providerName`      | string  | Yes       | `'Symphony'`      |
+| `id`  | object  | Yes       | `{ streamId: 'j75xqXy25NBOdacUI3FNBH', anyOtherKey: 'abcdef'}` |
+| `url` | string  | No       | `'http://symphony.com/ref/room/j75xqXy25NBOdacUI3FNBH___pqSsuJRdA'`      |
+| `name` | string  | No       | `'My new room'`      |
+
+The `url` is an universal url to access to the room. It could be opened from a browser, a mobile app, etc...
```

Comment:
```suggestion
The `url` is a universal url to access to the room. It could be opened from a browser, a mobile app, etc...
```

Had to double check for this one, but the internet consensus seems to be 'a universal'
https://www.pristineword.com/grammar-a-universal/